udata/features/territories/views.py

```diff
@@ -72,7 +72,7 @@ def compute_territory_dataset(territory, dataset, resource_id):
                      attachment_filename=attachment_filename)
 
 
-@blueprint.route('/territory/regions/', endpoint='home')
+@blueprint.route('/territoires/', endpoint='home')
```

Comment:
You're right. An alternative (in another PR) could be to use multiple decorators:
```python
@blueprint.route('/territoires/', endpoint='home-fr')
@blueprint.route('/territories/', endpoint='home-en')
```